templates/python/custom-copilot-basic/.vscode/launch.json.tpl

```diff
@@ -7,7 +7,7 @@
       "request": "launch",
       "url": "https://teams.microsoft.com/l/app/${{TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&${account-hint}",
       "presentation": {
-        "group": "1-Teams",
```

Comment:
whether local debug or test tool is the 1st choice depends on the flag {{#enableTestToolByDefault}}, which depends on if the TTK is a pre-release or a stable release. 
I think it's ok to merge this first. Let's make the change in another PR.

Reply:
I'll check with you offline. Thanks.